ceph.spec.in

```diff
@@ -356,6 +356,7 @@ BuildRequires:	lz4-devel >= 1.7
 %if 0%{?fedora} || 0%{?rhel}
 BuildRequires:	libtool-ltdl-devel
 BuildRequires:	ninja-build
+BuildRequires:	podman
```

Comment:
Leap and tumbleweed both have podman so maybe move this to a less restrictive section?

Reply:
I am going to drop this part. as I haven't tested this setting.